src/dev/sass/build_sass.js

```diff
@@ -63,5 +52,41 @@ export async function buildSass({ log, kibanaDir }) {
     throw createFailError(`${message} on line ${line} of ${file}`);
   }
 
-  log.success('%d scss bundles created', bundleCount);
+  log.success('%d scss bundles %s', bundleCount, watch ? 'rebuilt' : 'created');
+};
+
+export async function buildSass({ log, kibanaDir, watch }) {
+  log.info('running plugin discovery in', kibanaDir);
+
+  const scanDirs = [resolve(kibanaDir, 'src/legacy/core_plugins')];
+  const paths = [resolve(kibanaDir, 'x-pack'), resolve(kibanaDir, 'node_modules/x-pack')];
+  const { spec$ } = findPluginSpecs({ plugins: { scanDirs, paths } });
+  const enabledPlugins = await spec$.pipe(toArray()).toPromise();
+  const uiExports = collectUiExports(enabledPlugins);
+  const { styleSheetPaths } = uiExports;
+
+  log.info('%s %d styleSheetPaths', watch ? 'watching' : 'found', styleSheetPaths.length);
+  log.verbose(styleSheetPaths);
+
+  if (watch) {
+    const debouncedBuild = debounce(async path => {
```

Comment:
Was there supposed to be a wait defined for the debounce here?

Reply:
Hmm... I'm not sure.  I saw this methodology elsewhere in Kibana for a watch flag, so I took it as the best approach.